### ISSUE TITLE
Broken stylesSet since v4.7.2

### ISSUE THREAD

**Issue by trsteel88** [ckeditor/ckeditor-dev#1108]
Since v4.7.2 styleSets I use a broken.

```
CKEDITOR.editorConfig = function( config ) {
    config.stylesSet = [
        { name: 'Intro', element: 'p', attributes: {'class': 'intro'} },
        { name: 'Primary button', element: 'a', attributes: {'class': 'btn btn-primary'} }
    ];
};
```

In the above example, the primary button doesn't show in the styles dropdown.

Is anyone else having a similar issue?

**Comment by trsteel88** [comment]
Fyi, the intro style appears. However, any styles for a links fail.

**Comment by NickWilde1990** [comment]
Might be https://github.com/ckeditor/ckeditor-dev/issues/862?

**Comment by msamsel** [comment]
Duplicated #862
bug is fixed and will be available from 4.8.0.
